website/app/styles/global/index.scss

```diff
@@ -1,3 +1,4 @@
 // GLOBAL STYLES
 
 @import "./link";
+@import "./sr-only";
```

Comment:
probably for consistency with what we're doing in the HDS components, I would move this to the `website/app/styles/app.scss` file (it's a global declaration, and is special)

Reply:
cool, moved it to `app.scss`